theme/src/components/code.js

```diff
@@ -41,6 +41,7 @@ function Code({className, children, live, noinline}) {
                     key={key}
                     fontFamily="mono"
                     fontSize={1}
+                    tabIndex={0}
```

Comment:
This change is making the screen reader weird.

Reply:
But adding tabIndex to the pre tag works fine but since it is inside external component we won't be able to make such change. 😞 

<img width="505" alt="image" src="https://user-images.githubusercontent.com/8944724/199587933-38fdbd53-a919-44e6-8b88-62736e736842.png">

Reply:
@amit-avit exactly, it's a 3rd part component... Can you please elaborate on what is exactly weird with the screen reader?

Reply:
If u see in the attached video for the focusable box, voiceOver reads the content 3 times.

https://user-images.githubusercontent.com/8944724/199778850-832936f4-f803-4acb-ba50-b8eff2152935.mov

Reply:
Unfortunately I cannot reproduce it neither in Safari nor in Chrome, on MacOS. I have a theory, at it looks like it judjing by your video that you have some additional Chrome extension that causes such behavior. Can it be the case? On clean chrome it reads exactly one time. 

P.S. What app are you using to screen record on your MacOS? cmd+shift+5 doesn't work properly with VoiceOver enabled for me...

Reply:
I'm able to repro in chrome but not in safari. I hope your are trying to navigate using ctrl+opt+righArrow and not tab.

Reply:
Yep, that combo helped to reproduce. BTW is it the correct way to navigate for a11y testing?

Reply:
Ok, thanks, I will use it to test the fix to this triple announcement.

Reply:
Hey @amit-avit I just got back on this today, and seem to have fixed it, can you please confirm?

Reply:
Yes this is fixed.